src/main/java/seedu/duke/logic/commands/TutorialCommand.java

```diff
@@ -0,0 +1,5 @@
+package seedu.duke.logic.commands;
+
+public interface TutorialCommand {
```

Comment:
Interface with only a constant?

Reply:
Couldn't think of a good way to generalise the tutorial command whilst also splitting between the two kinds of circuits. Since the both the classes already inherited from BooleanCommand/CircuitCommand I thought the only way we could define the characteristics of a TutorialCommand and implement was through an interface (oof)

Reply:
@hughjazzman  I might be able to add a function declaration here (startTutorial()), I guess I add it in another PR, or ?